benchmarks/benchmark_segmentation.py

```diff
@@ -1,23 +1,40 @@
 # See "Writing benchmarks" in the asv docs for more information.
 import numpy as np
+from numpy.lib import NumpyVersion as Version
+import skimage
 from skimage import segmentation
 
 
 class SegmentationSuite:
     """Benchmark for segmentation routines in scikit-image."""
     def setup(self):
-        self.image = np.random.random((400, 400, 100))
-        self.image[:200, :200, :] += 1
-        self.image[300:, 300:, :] += 0.5
-        self.msk = np.zeros((400, 400, 100))
+        self.image = np.random.random((200, 200, 100))
+        self.image[:100, :100, :] += 1
+        self.image[150:, 150:, :] += 0.5
+        self.msk = np.zeros((200, 200, 100))
```

Comment:
Why reduce this?

Reply:
The masked case takes ~1 minute and can time out seen in https://github.com/scikit-image/scikit-image/pull/3120#issuecomment-670739755. We don't strictly need to change this in this PR, though. I can revert to the original size if you prefer.

Reply:
Also, the changes proposed in #4903 would reduce the run time for the original size to around 20 seconds.

Reply:
(The 60 s and 20 s quoted above are for a single repetition, so I assume it takes substantially longer than that when `asv dev` or `asv run --quick` isn't being used.